scout/server/blueprints/variants/controllers.py

```diff
@@ -936,7 +936,7 @@ def generate(header, lines):
     ]
 
     export_lines = []
-    for variant in variant_objs:
+    for variant in variant_objs.limit(500):
```

Comment:
Ahhh, that's because STRs use a function apart to export variants. I think it would be better for maintanance to create a constant up in this file, like MAX_EXPORTED_VARIANTS = 500, and then use it here and in the function `download_variants`, that is down at line 977